.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Cross-Platform CI
```

Comment:
It'd be nice with some rendezvous inspiration here, like *Tokyo Nights*? 👀

Reply:
The contributors will see this when opening PRs. IMO it could be misleading, but I have no problem updating it.